docs/deployment/README.md

```diff
@@ -153,48 +151,35 @@ Cumulus supports operation within a VPC, but you will need to separately create:
 - Security group
 - VPC endpoints for the various services used by Cumulus if you wish to route traffic through the VPC
 
-These resources only need to be created once per AWS account and their ids will be used to configure your Terraform deployment.
+These resources only need to be created once per AWS account and their IDs will be used to configure your Terraform deployment.
 
 #### Elasticsearch in a VPC
 
-Amazon Elasticsearch Service does not use a VPC Endpoint. To use ES within a VPC, before deploying run:
+Amazon Elasticsearch Service (ES) does not use a VPC Endpoint. To use ES within a VPC, before deploying run:
 
 ```shell
 aws iam create-service-linked-role --aws-service-name es.amazonaws.com
 ```
 
 This operation only needs to be done once per account, but it must be done for both NGAP and regular AWS environments.
 
-### Look up ECS-optimized AMI (DEPRECATED)
-
```

Comment:
This is still helpful for existing instance

Reply:
@jennyhliu I will add these sections back in for now. We can discuss at later time about handling archived and deprecated information. Thank you.